engine/src/gfx/nav/navitemstodraw.h

```diff
@@ -25,14 +25,14 @@ void NavigationSystem::DrawCursor( float x, float y, float wid, float hei, const
         GFXBlendMode( SRCALPHA, INVSRCALPHA );
 
         const float verts[8 * 3] = {
-            x,          y,          0,
-            x,          y-hei,      0,
-            x,          y,          0,
-            x+wid,      y-0.75*hei, 0,
-            x,          y-hei,      0,
-            x+0.35*wid, y-0.6*hei,  0,
-            x+0.35*wid, y-0.6*hei,  0,
-            x+wid,      y-0.75*hei, 0,
+            x,                              y,                              0,
+            x,                              static_cast<float>(y-hei),      0,
+            x,                              y,                              0,
+            static_cast<float>(x+wid),      static_cast<float>(y-0.75*hei), 0,
+            x,                              y-hei,                          0,
+            static_cast<float>(x+0.35*wid), static_cast<float>(y-0.6*hei),  0,
+            static_cast<float>(x+0.35*wid), static_cast<float>(y-0.6*hei),  0,
+            static_cast<float>(x+wid),      static_cast<float>(y-0.75*hei), 0,
```

Comment:
Is this all space aligned or tab aligned?
Let's make sure it's space aligned.

Reply:
Everything is space aligned, I have rechecked just to make sure.